aiohttp/_websocket/reader_c.py

```diff
@@ -0,0 +1 @@
+reader_py.py
```

Comment:
I was a little worried about adding a symlink, but I did a build with `python -m build` and it includes a copy of it so it works just fine

Reply:
I went ahead and added coverage here since we needed to fix the incoming type for `feed_data`